lib/on_membership_event.js

```diff
@@ -137,10 +142,12 @@ function createUpdatedHandlerForRing(ringpop) {
             var update = updates[i];
             switch (update.status) {
                 case Member.Status.alive:
+                // TODO: Faulty should be here
```

Comment:
It is not `faulty` that should be here, but yes the `suspect` case should also add to the hash ring.